docs: translation process modern (#3948)
JesusFilm/core: apps/docs/docs/04-engineering-practices/03-internationalization/02-implementation/02-supported-languages.md

# Supported Languages

> These are the languages currently supported in Core. Do note that not all the languages listed have been translated yet. Please take the time to look into what's been translated in Crowdin.

Reviewer: Below are the languages currently supported in Core. Note that not all the languages listed have translations yet. Please take the time to look into what has been translated in Crowdin.
```suggestion
Below are the languages currently supported in Core. Note that not all of the languages listed have translations yet. Please take the time to look into what has been translated in Crowdin.
```

| Chinese    | Traditional, China | zh-Hant-TW          | zh-Hant-TW              |
| English    | Default Variant    | en                  | en                      |
| French     | France             | fr                  | fr-FR                   |
| Hindi      | India              | hi                  | hi-IN                   |
| Indonesian | Indonesia          | id                  | id-ID                   |
| Japanese   | Japan              | ja                  | ja-JP                   |
| Korean     | Korea              | ko                  | ko-KR                   |
| Russian    | Russia             | ru                  | ru-RU                   |
| Spanish    | Spain              | es                  | es-ES                   |
| Tagalog    | Philippines        | tl                  | tl-PH                   |
| Thai       | Thailand           | th                  | th-TH                   |
| Turkish    | Turkey             | tr                  | tr-TR                   |
| Urdu       | Pakistan           | ur                  | ur-PK                   |
| Vietnamese | Vietnam            | vi                  | vi-VN                   |

> Note: While most languages in the locales list use two-letter codes, some variants like Chinese Traditional (`zh-Hant-TW`) require more specific codes to properly support different versions of the same language using the format language-SCRIPT-REGION format.

Reviewer: ```suggestion
Note: While most languages in the locales list use two-letter codes, some variants such as Traditional Chinese (`zh-Hant-TW`) require more specific codes to properly support different versions of the same language using the format (`language-SCRIPT-REGION`) format.
```
